specification/synapse/resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/examples/CreateOrUpdateWorkspace.json

```diff
@@ -26,7 +26,16 @@
         "sqlAdministratorLogin": "login",
         "sqlAdministratorLoginPassword": "password",
         "managedVirtualNetwork": "default",
-        "managedResourceGroupName": "workspaceManagedResourceGroupUnique"
+        "managedResourceGroupName": "workspaceManagedResourceGroupUnique",
+        "workspaceRepositoryConfiguration": {
```

Comment:
this should also be specified in the response(s)

Reply:
this is an async API call git configuration details won't exist in this response.